src/components/shared/SearchCard/SearchCardFooter.js

```diff
@@ -1,19 +1,19 @@
 import { useRouter } from 'next/router'
 import { Box } from 'grommet'
-import { useSearchManager } from 'hooks/useSearchManager'
 import formatURLString from 'helpers/formatURLString'
 import gtag from 'analytics/gtag'
 import { Button } from 'components/shared/Button'
 
 export const SearchCardFooter = ({ experiment }) => {
   const { accession_code: accessionCode, title } = experiment
   const { push } = useRouter()
-  const { search, setSearch } = useSearchManager()
 
   const handleClick = () => {
-    setSearch({ ...search, ref: 'search' })
     gtag.trackExperimentPageClick(SearchCardFooter)
-    push(`/experiments/${accessionCode}/${formatURLString(title)}`)
+    push({
+      pathname: `/experiments/${accessionCode}/${formatURLString(title)}`,
+      query: { ref: 'view-samples' }
```

Comment:
Removed the use of `useSearchManager` for the search page reference and adjusted `gtag.trackLink` accordingly.